gridcapa-core-cc-post-processing-app/src/main/java/com/farao_community/farao/core_cc_post_processing/app/services/CoreCCMetadataGenerator.java

```diff
@@ -50,11 +54,19 @@ private static MultiKeyMap structureDataFromTask(List<CoreCCMetadata> metadataLi
         data.put(RAO_COMPUTATION_STATUS, macroMetada.getTimeInterval(), macroMetada.getStatus());
         data.put(RAO_START_TIME, macroMetada.getTimeInterval(), macroMetada.getComputationStartInstant());
         data.put(RAO_END_TIME, macroMetada.getTimeInterval(), macroMetada.getComputationEndInstant());
-        data.put(RAO_COMPUTATION_TIME, macroMetada.getTimeInterval(), String.valueOf(ChronoUnit.MINUTES.between(Instant.parse(macroMetada.getComputationStartInstant()), Instant.parse(macroMetada.getComputationEndInstant()))));
+        if (StringUtils.isBlank(macroMetada.getComputationStartInstant()) || StringUtils.isBlank(macroMetada.getComputationEndInstant())) {
```

Comment:
Perhaps create a shared method for getting computation time, avoiding repeating same test twice ?

Reply:
done